app/views/layouts/application.html.erb

```diff
@@ -30,6 +30,7 @@
     <% end %>
     <%= yield :javascript %>
     <%= javascript_include_tag "jquery.simulate" if Rails.env.test? %>
+    <!--[if lt IE 10]> <%= javascript_include_tag "flexibility" %> <![endif]-->
```

Comment:
I don't have IE10 to test. But Flex is supported by IE10 by default. So I think it should behave like IE11.